src/core/reactor.cc

```diff
@@ -4369,6 +4373,7 @@ void smp::configure(const smp_options& smp_opts, const reactor_options& reactor_
     reactor_config reactor_cfg;
     reactor_cfg.auto_handle_sigint_sigterm = reactor_opts._auto_handle_sigint_sigterm;
     reactor_cfg.max_networking_aio_io_control_blocks = adjust_max_networking_aio_io_control_blocks(reactor_opts.max_networking_io_control_blocks.get_value());
+    reactor_cfg.using_seastar_allocator = (smp_opts.memory_allocator == memory_allocator::seastar);
```

Comment:
In debug mode we don't use the seastar allocator, does smp_opts.memory_allocator reflect that?

Reply:
ok. but all CI checks failed.


I guess the cause is, once you call anything in memory.cc, it initializes thread-local variables and breaks things.